cobrexa.def

```diff
@@ -1,11 +1,11 @@
-Bootstrap: library
+BootStrap: library
```

Comment:
Is this change important?

Reply:
not sure, docs are using BootStrap and this remained here from trying to fix another issue. Will revert